### DISCUSSION TITLE
HTML Site Analytics

### DISCUSSION THREAD

**Discussion by umutguden** [vercel/community#5247] [Help]
### Summary

I use vercel for my one-page html site and I want the analytics. Is it possible via Vercel Analytics? Thanks.

### Steps to Reproduce

Dashboard > Project > Analytics

**Comment by amyegan** [comment]
Hi @ichbinheimdall! You can find instructions for how to do this in our [Vercel Web Analytics Quickstart](https://vercel.com/docs/analytics/quickstart#) documentation. To see the steps for an HTML site, you'll need to choose `HTML` in the framework selection box near the top of the page

<img width="1013" alt="screenshot showing HTML as the selected framework" src="https://github.com/vercel/community/assets/8247105/51a5f0fd-96dc-41c4-a3d8-e3cc59c4e555">
